### PR TITLE
Make debugging easier

### PR DESCRIPTION
**Pull request by hjmcnew**
Restructure the code a bit to allow for easier configuration around debugging. Also clean up the install documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,11 +17,13 @@ There are two ways to install:
      gpio_pin: D0
      
      # (Optional) Path to the leapmmw_sensor.h file relative to your esphome configuration directory.
-     header_file: leapmmw_sensor.h
+     # header_file: leapmmw_sensor.h
```
Comment (crlogic) [inline README.md:20]: Are we sure we want this commented out as the default?  I am not sure people understand the correlation between this line and being able to make changes to sensor settings.  Might be good to expand on that.

Comment (hjmcnew) [reply README.md:20]: Happy to clarify the comment but in terms of functionality it is no change. The file path is included in the substitutions in the package. This just allows one to override the location in their config (if they haven’t saved it in the top level esphome directory).

Comment (crlogic) [reply README.md:20]: ah, that makes sense.  Still getting a handle on this new fancy packaging :)

**Comment by crlogic** [comment]
LGTM

**Comment by hjmcnew** [comment]
@crlogic Would appreciate a quick re-review. I re-added the uart config to the leapmmw.yml file to prevent a breaking change for folks who deployed with `file:` and won't have picked up the `files: []` yet. I'll cut a new release after this gets merged.

**Comment by crlogic** [comment]
Test #1 - build sensor on release changed to make-debug-easy with debug enabled.
```
packages:
  remote_package:
    refresh: 0s
    url: https://github.com/hjmcnew/esphome-hs2xx3a-custom-component
    ref: make-debug-easy
    files: [packages/uart_debug.yml, packages/leapmmw_sensor.yml]
```
Confirmed the receipt of debug logging and that LED controls functioned on and off while displaying debug data.

Test #2 - build sensor on release changed to make-debug-easy with debug disabled.
```
packages:
  remote_package:
    refresh: 0s
    url: https://github.com/hjmcnew/esphome-hs2xx3a-custom-component
    ref: make-debug-easy
    files: [packages/uart.yml, packages/leapmmw_sensor.yml]
```
Confirmed no debug logging and that LED controls functioned on and off.

**Comment by hjmcnew** [comment]
> Test #1 - build sensor on release changed to make-debug-easy with debug enabled.
> 
> ```
> packages:
>   remote_package:
>     refresh: 0s
>     url: https://github.com/hjmcnew/esphome-hs2xx3a-custom-component
>     ref: make-debug-easy
>     files: [packages/uart_debug.yml, packages/leapmmw_sensor.yml]
> ```
> 
> Confirmed the receipt of debug logging and that LED controls functioned on and off while displaying debug data.
> 
> Test #2 - build sensor on release changed to make-debug-easy with debug disabled.
> 
> ```
> packages:
>   remote_package:
>     refresh: 0s
>     url: https://github.com/hjmcnew/esphome-hs2xx3a-custom-component
>     ref: make-debug-easy
>     files: [packages/uart.yml, packages/leapmmw_sensor.yml]
> ```
> 
> Confirmed no debug logging and that LED controls functioned on and off.

So, uh, is that a 👍?
😀

**Comment by crlogic** [comment]
You got it ;)

**Pull request merged**